Applications/Advanced/HandEyeCalibration/README.md

```diff
@@ -0,0 +1,31 @@
+## Hand Eye Calibration
```

Comment:
Top level heading should only have one #

I use the markdownlint plugin in VS code to help with consistent markup use and formatting